src/utility/index.jsx

```diff
@@ -18,6 +18,14 @@ import subTextStyle from 'components/Visualizations/Table/subText.css';
 import findLast from 'lodash.findlast';
 import _ from 'lodash/fp';
 import util from 'util';
+// import SvgIcon from 'material-ui/SvgIcon';
+import SocialPeople from 'material-ui/svg-icons/social/people';
+import SocialPerson from 'material-ui/svg-icons/social/person';
+
+const iconStyles = {
+  marginBottom: -5,
```

Comment:
if u'll set width & height to 18px, u can delete negative margin bottom (I'm not a big fan of negative margins 😛) and use `vertical-align`